kasa/iot/iotstrip.py

```diff
@@ -32,7 +33,7 @@ def merge_sums(dicts):
     return total_dict
 
 
-class IotStrip(IotDevice):
+class IotStrip(IotDevice, Strip):
```

Comment:
We could probably get rid of `Strip` by handling powerstrips as `Plug`s that just happen to have children? :-)

Reply:
Done